Add text and examples for phpcbf and php-cs-fixer

ericpoe opened the pull request:
After the obsolete `php.tools` information was removed, the surrounding text seemed disjointed. This PR rearranges that text, adds information about `phpcbf`, and adds examples for both `phpcbf` and `php-cs-fixer`.

I tried to keep the structure the same between the examples for `phpcs`, `phpcbf`, and `php-cs-fixer`.

ericpoe committed:
Add text and examples for phpcbf and php-cs-fixer

ericpoe commented:
This fixes #669

naveenvalecha commented:
The title needs updation for clean git history as the PR also adding the examples of phpcs and phpcbf.

ericpoe commented:
I do not understand your request @naveenvalecha. This PR keeps the the existing example of phpcs and adds text and examples for `phpcbf` and `php-cs-fixer`. The title of the PR reflects that.

Also, the title of this PR has no bearing on git-history. The git message within the PR is accurate, so that shouldn't have any bearing on the git-history either.

naveenvalecha commented:
Looks good.
P.S..  Github should provide the edit option to comments when opening site on mobile.

ericpoe committed:
Remove awkward clause.

The phrase about how well tested this library is  seems out of place
for a library that is compared to others in the same section. One would
assume that that all of these are well tested.

ericpoe commented:
This PR is ready for review. Thanks!
